CREDITS

```diff
@@ -991,3 +991,8 @@ N: Enzo Carozza
 S: uni
 E: dudetank4@gmail.com
 D: CREDITS-MUSIC
+
+S: Magnolia
+D: sprites
+
+
```

Comment:
There's some extra whitespace at the end of this file now. No big deal, and we'll probably have opportunities to address it before the release.